DSCResources/MSFT_xDNSServerAddress/MSFT_xDNSServerAddress.psm1

```diff
@@ -47,36 +55,46 @@ function Get-TargetResource
     Write-Verbose -Message ( @( "$($MyInvocation.MyCommand): "
         $($LocalizedData.GettingDNSServerAddressesMessage)
         ) -join '')
+    
+    # obtain all the interface addresses
+    $dnsClientServerAddresses = Get-NetAdapter -Name $InterfaceAlias `
```

Comment:
Could you help me understand this change better? I'm not sure I'm clear on why its needed? Also when testing it on my machine I get an error when running this command.

````powershell
[4:29:27 PM] [pr-57] [0057] » Get-TargetResource -Address '127.0.0.1' -InterfaceAlias '*Ethernet*' -AddressFamily 'IPv4'
Get-DnsClientServerAddress : No matching MSFT_DNSClientServerAddress objects found by CIM query for instances of the ROOT/StandardCimv2/MSFT_DNSClientServerAddress class on the  CIM server: SELECT * FROM MSFT_DNSClientServerAddress
WHERE ((InterfaceAlias LIKE 'MSFT[_]NetAdapter (CreationClassName = "MSFT[_]NetAdapter", DeviceID = "{EB44F4BF-30C1-401E-821A-AAAF8229A5E6}", SystemCreationClassName = "CIM[_]NetworkPort", SystemName =
"<ComputerName>)')) AND ((AddressFamily = 2)). Verify query parameters and retry.
At C:\GitHub\tysonjhayes\xNetworking\DSCResources\MSFT_xDNSServerAddress\MSFT_xDNSServerAddress.psm1:61 char:24
+     | ForEach-Object { Get-DnsClientServerAddress -InterfaceAlias $_  -AddressFa ...
+                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (MSFT_DNSClientServerAddress:String) [Get-DnsClientServerAddress], CimJobException
    + FullyQualifiedErrorId : CmdletizationQuery_NotFound,Get-DnsClientServerAddress
````

I get the same thing when I don't try a wild card for the interface alias. Where the previous comamnd worked fine.